data/json/recipes/other/parts.json

```
@@ -414,9 +414,9 @@
     "reversible": true,
     "decomp_learn": 2,
     "book_learn": [ [ "manual_electronics", 2 ], [ "mag_electronics", 2 ], [ "manual_mechanics", 2 ] ],
-    "using": [ [ "soldering_standard", 5 ], [ "steel_tiny", 1 ] ],
+    "using": [ [ "soldering_standard", 5 ], [ "steel_tiny", 1 ], [ "welding_standard", 2 ] ],
     "qualities": [ { "id": "HAMMER", "level": 1 }, { "id": "SCREW_FINE", "level": 1 } ],
-    "components": [ [ [ "cable", 3 ] ], [ [ "e_scrap", 3 ] ] ]
+    "components": [ [ [ "cable", 5 ] ], [ [ "e_scrap", 3 ] ], [ [ "pipe", 1 ] ], [ [ "bearing", 4 ] ] ]
```

```suggestion
    "components": [ [ [ "cable", 5 ] ], [ [ "e_scrap", 3 ] ], [ [ "scrap", 1 ] ], [ [ "bearing", 4 ] ] ]
```
The dda pipe is significantly longer and larger than these motors I believe.

---

I guess my issue is that maybe you could use a pipe to make this motor but you'd be unable to reverse it and get the pipe back

---

That makes sense